applications/hotornot/src/hon_util.erl

```diff
@@ -161,14 +185,23 @@ sort_rate(RateA, RateB) ->
             PrefixA > PrefixB
     end.
 
+-spec sort_rate_by_cost(kz_json:object(), kz_json:object()) -> boolean().
+sort_rate_by_cost(RateA, RateB) ->
+    PrefixA = byte_size(kz_json:get_binary_value(<<"prefix">>, RateA)),
+    PrefixB = byte_size(kz_json:get_binary_value(<<"prefix">>, RateB)),
+
+    case PrefixA =:= PrefixB of
+        'true' ->
+            kz_json:get_float_value(<<"rate_cost">>, RateA, 0) >
+                kz_json:get_float_value(<<"rate_cost">>, RateB, 0);
```

Comment:
I'm not sure I follow this reasoning - if "rate_cost" is undefined in either, it will default to 0. The comparison can only ever be a float value or 0.